tests/e2e/ccruntime-ssh-demo_tests.bats

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bats
```

Comment:
Seems like maybe this should go in the tests repo? I guess this is fine for the short term.